components/style/values/specified/length.rs

```diff
@@ -1226,3 +1227,97 @@ impl LengthOrNumber {
         }
     }
 }
+
+/// A value suitable for a `min-width` or `min-height` property.
+/// Unlike `max-width` or `max-height` properties, a MinLength can be
+/// `auto`, and cannot be `none`.
+#[derive(Debug, Clone, PartialEq)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+#[allow(missing_docs)]
+pub enum MinLength {
```

Comment:
Is there any semantic difference between `MinLength` and `MaxLength`? If not I'd prefer for these to be the same type, called `ExtremumLength`, and what is currently `ExtremumLength` can become `ExtremumLengthKeyword`